docs/coding-guidelines/updating-ref-source.md

```diff
@@ -2,7 +2,7 @@ This document provides the steps you need to take to update the reference assemb
 
 ## For most assemblies within libraries
 
-1. Implement the API in the source assembly and [build it](../workflow/building/libraries/README.md#building-individual-libraries).
+1. Implement the API in the source assembly and [build it](../workflow/building/libraries/README.md#building-individual-libraries). Note that when adding new public types, this might fail with a `TypeMustExist` error. The deadlock can be worked around by disabling the `RunApiCompat` property: `dotnet msbuild /p:RunApiCompat=false`.
```

Comment:
Please let's use `dotnet build` as it also restores incrementally which is now required for our ref & impl assemblies.

```suggestion
1. Implement the API in the source assembly and [build it](../workflow/building/libraries/README.md#building-individual-libraries). Note that when adding new public types, this might fail with a `TypeMustExist` error. The deadlock can be worked around by disabling the `RunApiCompat` property: `dotnet build /p:RunApiCompat=false`.
```

Reply:
Note that other lines in the same document mention the msbuild command, so probably a good idea to enforce consistency here.

Reply:
Thanks, will take care of that.